functional-tests/config.js

```diff
@@ -13,12 +13,12 @@ let conf = {
     timeout: 50000,
     compilers: ['js:babel-core/register'],
   },
-  baseUrl: 'http://localhost:9000',
+  baseUrl: 'http://10.200.10.1:9000',
```

Comment:
It's the one of your HOST machine when the browsers inside docker will try to access the test web server app.

It would have been better to keep "localhost" but docker cannot do that on osx right now sadly.